boards/arduino-mega2560/examples/mega2560-adc.rs

```diff
@@ -23,7 +23,7 @@ fn main() -> ! {
     );
     let mut delay = arduino_mega2560::Delay::new();
 
-    let mut serial = arduino_mega2560::Serial::new(
+    let mut serial = arduino_mega2560::usart::Usart0::new(
```

Comment:
@Rahix can we simplify this? Maybe we can redefine new in board to skip first attribute?

Reply:
I don't think so.  As I said in the other comment, I'd keep the `Serial` alias so the examples don't need any changing.